Registry/Registry.EM_COMMON

```diff
@@ -486,8 +486,11 @@ state   real    dfi_qh         ikjftb   dfi_moist       1         -     \
    rusdf=(bdy_interp:dt)  "DFI_QHAIL"        "Hail mixing ratio"             "kg kg-1"
 state    real   qvold            ikj    misc        1         -     rdu      "QVOLD"          "Water vapor mixing ratio, old time step"      "kg kg-1"
 state    real   rimi             ikj    misc        1         -     irh      "RIMI"           "riming intensity"  "fraction"
-state    real   qnwfa2d          ij     misc        1         -     i014rhdu "QNWFA2D"        "Surface aerosol number conc emission"  "kg-1 s-1"
-state    real   qnifa2d          ij     misc        1         -     i014rhdu "QNIFA2D"        "Surface dust number conc emission"  "kg-1 s-1"
+state    real   qnwfa2d          ij     misc        1         -    i014{17}rhdu "QNWFA2D"       "Surface aerosol number conc emission"  "kg-1 s-1"
+state    real   qnifa2d          ij     misc        1         -    i014{17}rhdu "QNIFA2D"       "Surface dust number conc emission"  "kg-1 s-1"
+state    real   qnbca2d          ij     misc        1         -    i014{17}rhdu "QNBCA2D"       "Surface black carbon number conc emission"  "kg-1 s-1"
+state    real   qnocbb2d         ij     misc        1         -    i014{17}rhdu "QNOCBB2D"      "Surface organic carbon biomass burning number conc emission"  "kg-1 s-1"
+state    real   qnbcbb2d         ij     misc        1         -    i014{17}rhdu "QNBCBB2D"      "Surface black carbon biomass burning number conc emission"  "kg-1 s-1"
```

Comment:
@twjuliano 
Since these are i4 and i1, that means that we need these variables in WPS.  Have mods to the WPS tables been provided, and where is this global static data?

Reply:
@davegill Yes, I have added the mods for METGRID.TBL to the previous WPS PR (https://github.com/wrf-model/WPS/pull/190). The update global static file (GOCART climatology) is available in the Google Drive link posted in this PR as well as the WPS PR (https://drive.google.com/file/d/1BYflyu65kP5giRYbTzKo6y4iSnTfb1Fw/view?usp=sharing).

Reply:
@twjuliano If you place these fields in stream 17, perhaps you should remove '4' from the group.

Reply:
@weiwangncar OK, I can do that. I maintained what was there previously just in case...

Reply:
@twjuliano When you remove stream 4 from these lines, perhaps you could also line up the columns? This is just for easy viewing and tidiness of the code.

Reply:
@weiwangncar Sure thing, will do.